### PR TITLE
October news

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: content/blog/2024-10-23-news-october-2024/index.Rmd [hunk 1]
```diff
@@ -0,0 +1,620 @@
...
+With this $340K grant, we’re planning to launch a Spanish-language version of our Champions Program, along with other new initiatives to make sustainable software development more accessible to researchers across the region. 
+Read more in our [announcement](/blog/2024/10/10/czi-latam-grant/).
+
+### R-Universe interactive tour
+
+Each package page on R-universe now features a guided tour built with intro.js, so that you might get an overview of all information presented on package pages.
+To start the tour, click on the "Website Tour" button on the navbar -- note that it does not work on mobile devices.
+Example: [xml2 package](https://r-lib.r-universe.dev/xml2).
+
+### The rOpenSci community at upcoming events
```
Comment (steffilazerte) [inline content/blog/2024-10-23-news-october-2024/index.Rmd:59]: Should we add some more details of all the other things going on during LatinR? Or at least link to the event: https://ropensci.org/events/latinr-2024/

File: content/blog/2024-10-23-news-october-2024/index.Rmd [hunk 2]
```diff
@@ -0,0 +1,620 @@
...
+Join us for social coworking & office hours monthly on first Tuesdays! 
+Hosted by Steffi LaZerte and various community hosts. 
+Everyone welcome. 
+No RSVP needed. 
+Consult our [Events](/events) page to find your local time and how to join.
+
+- [Tuesday, November 5th, 09:00 Americas Pacific (17:00 UTC)](/events/coworking-2024-11), "Getting to know Openscapes" with cohost [Stefanie Butland](/author/stefanie-butland) and [Steffi LaZerte](/author/steffi-lazerte).
+    - Look up [Openscapes](https://openscapes.org/)
+    - Chat with our cohost to learn more about the Openscapes organization, and how they use R to automate creating open education infrastructure
+
```
Comment (steffilazerte) [inline content/blog/2024-10-23-news-october-2024/index.Rmd:87]:
```suggestion

- [Tuesday, December 3rd, 09:00 Australia Western (01:00 UTC)](/events/coworking-2024-12), "Getting Involved in the Antarctic/Southern Ocean rOpenSci Community" with cohosts [Michael Sumner](/author/michael-sumner ) and [Ben Raymond](/author/ben-raymond), and [Steffi LaZerte](/author/steffi-lazerte).
    - [Explore rOpenSci R packages](/blog/2018/11/13/antarctic/#demo) supporting Antarctic and Southern Ocean Research
    - Chat with our cohosts to find out how you can be involved in this community

```

**Review by steffilazerte — COMMENTED** [review]
I just sneaked this in so reviewing it got bumped up the list to a 'this-week' priority 😉 

Looks good, and I added the coworking event (although it hasn't been reviewed by the community hosts yet, so I may need to tweak it if they have suggestions).

Thanks!

* [X] I ran `roblog::ro_lint_md()` on index.md
* [X] I ran `roblog::ro_check_urls()` on index.md
* [X] I ran a spell-check on index.md

**Review by yabellini — COMMENTED** [review]
I added the community call and more detail about LatinR.

File: content/blog/2024-10-23-news-october-2024/index.Rmd (hunk 1 above)
Comment (maelle) [reply content/blog/2024-10-23-news-october-2024/index.Rmd:59]: done by @yabellini

File: content/blog/2024-10-23-news-october-2024/index.Rmd [hunk 3]
```diff
@@ -0,0 +1,646 @@
...
+🕓 Wednesday, 06 November 2024 13:00 UTC 
+With [Pedro Faria](/author/pedro-faria/), [Beatriz Milz](/author/beatriz-milz/), [Francesca Belem Lopes Palmeira](/author/francesca-belem-lopes-palmeira/).
+
+The R Community is carrying out several translations into Portuguese of different resources: books, packages and guides.  
+
+In this community call, three leaders from the Portuguese-speaking community will present their experiences leading and being part of these projects and share how we can participate in these efforts, including rOpenSci Developer Guide translation project.
+
+This community call will be followed by a [translaton at LatinR](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167), for those who want to contribute.
+
+### Coworking
```
Comment (steffilazerte) [inline content/blog/2024-10-23-news-october-2024/index.Rmd:96]:
```suggestion
This community call will be followed by a [translaton at LatinR](https://www.eventbrite.cl/e/traducao-hackathon-traslaton-ropensci-tickets-1045507530167), for those who want to contribute.

### Looking for Maintainers to Support First-Time Contributors
Making your first contribution to Open Source can be both empowering and intimidating. As such, we’re exited to announce a special series of activities to support first-time contributors to rOpenSci packages! 🎉

Are you an rOpenSci package maintainer who would like to help someone make their first contribution?

See our blog post for more details and how to sign up (by Nov 24): https://ropensci.org/blog/2024/10/22/first-time-contributions/

### Coworking
```

**Pull request merged**